tests/lib/rules/no-children-prop.js

```diff
@@ -15,6 +15,7 @@ var RuleTester = require('eslint').RuleTester;
 var parserOptions = {
   ecmaVersion: 6,
   ecmaFeatures: {
+    experimentalObjectRestSpread: true,
```

Comment:
I think you've got a valid point.

But I'm pretty big on code consistency, so I wonder if it would be better to do a separate PR with that change made in all of the cases like this.